schema/one-time-polled-screening.json

```diff
@@ -730,7 +730,7 @@ Note that, as this is a demo check, no credits will actually be deducted from th
                     },
                     "message": {
                         "type": "string",
-                        "description": "A message displayed in the PassFort portal when showing the check error."
+                        "description": "A message displayed in the Passfort portal when showing the check error."
```

Comment:
Can we change "when showing" to "to show"?

Reply:
"To show" may not be technically correct. The change on line 733 implies that the message shows the check error, but it would only be a description of the error and not the error itself. I found the initial description field definitions a bit unclear, so I reworked the error type and subtype descriptions to create further distinction between their respective decision fields. Let me know your thoughts: 

`            "Error": {
                "type": "object",
                "required": [
                    "type",
                    "message"
                ],
                "properties": {
                    "type": {},
                    "sub_type": {
                        "type": "string",
                        "enum": [
                            "UNSUPPORTED_COUNTRY",
                            "UNSUPPORTED_DOCUMENT_TYPE"
                        ],
                        "description": "Provides additional detail for the subtypes of certain errors."
                    },
                    "message": {
                        "type": "string",
                        "description": "A message displayed in the Passfort portal which gives further detail about the error type."
                    },
                    "data": {
                        "description": "Integration-specific structured error information that customers can use to get additional information about the error being reported."
                    }
                }
            },`

Reply:
Really good improvement! It's so much clearer now